### PR TITLE
(fix) - Update the getServiceStatus from screen to custom shell script

### PR DESCRIPTION
**Pull request by lolstring**
change to spaces (:

Update the status check.
Add a new shell script to check status.

### CODE REVIEW

File: mycroft@rahul.io/metadata.json [hunk 1]
```diff
@@ -1,15 +1,8 @@
...
  "settings-schema": "org.gnome.shell.extensions.mycroft",
- "shell-version": [
-  "3.14",
-  "3.16",
-  "3.18",
-  "3.20",
-  "3.22",
-  "3.24"
- ],
+ "shell-version": ["3.14", "3.16", "3.18", "3.20", "3.22", "3.24"],
```
Comment (lolstring) [inline mycroft@rahul.io/metadata.json]: Update this shell-version to 3.28

File: package.json [hunk 2]
```diff
@@ -5,17 +5,19 @@
...
   "devDependencies": {
     "del": "^2.2.2",
-    "eslint": "^3.9.1",
+    "eslint": "^3.19.0",
     "gulp": "^3.9.1",
     "gulp-eslint": "^3.0.1",
     "gulp-eslint-threshold": "^0.1.1",
-    "gulp-json-editor": "^2.2.1",
+    "gulp-json-editor": "^2.5.4",
+    "gulp-ruby-sass": "^4.0.0",
```
Comment (lolstring) [inline package.json]: ...remove this line

File: src/extension.js [hunk 3]
```diff
@@ -35,2121 +35,2328 @@ const MYCROFT_ANIMATION_STATUS_KEY = 'animation-status';
...
+    position_in_panel = this._position_in_panel;
+    core_location = this.core_location;
+    mycroft_is_install = this.mycroft_is_install;
+    install_type = this.install_type;
+    animation_status = this.animation_status;
+    this.setEventListeners();
+    if (mycroft_is_install) {
+      this.emitServiceStatus('install');
+    }
+    // _timeoutId = Mainloop.timeout_add(2000, Lang.bind(this, function() {
```
Comment (lolstring) [inline src/extension.js]: revert this.

**Comment by AIIX** [comment]
Looks good to me but what about supporting shell 3.30+ ?

**Comment by lolstring** [comment]
I need to flash a new distro on my computer to see what breaks before I rewrite/convert it, I am not sure if it's backward compatible.

**Pull request merged**